test-dotnet/action.yml

```diff
@@ -70,7 +70,7 @@ runs:
         for f in $(find . -name "*.Tests.csproj"); do echo "testing project $f" && \
           csprojName=${f##*/} && \
           projectName="${csprojName/.csproj/}" && \
-          dotnet test $f --configuration Debug --no-restore --logger GitHubActions --collect "XPlat Code Coverage" --results-directory "TestResults/$projectName"
+          dotnet test $f --configuration Debug --no-build --logger GitHubActions --collect "XPlat Code Coverage" --results-directory "TestResults/$projectName"
```

Comment:
Need to be both the --no-restore and --no-build

Reply:
Not according to the docs:

**--no-build**
Doesn't build the project before publishing. **It also implicitly sets the --no-restore flag.**

<https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-publish#:~:text=for%20each%20manifest.-,%2D%2Dno%2Dbuild,-Doesn%27t%20build%20the>